accord-core/src/main/java/accord/local/CommandStore.java

```diff
@@ -895,4 +938,11 @@ public NodeCommandStoreService node()
     {
         return node;
     }
+
+    /**
+     * Thrown when command store is not ready to serve the request for a given range.
+     */
+    public static class NotReadyException extends RuntimeException
```

Comment:
I think NotReady is perhaps the wrong name for this, since we will never be ready, unless I misunderstand the context. The point is we don't have the relevant log history anymore. The best we can ever hope to respond with is Vestigial. The only reason we might not convert immediately to Vestigial is that a transaction may only partially overlap the rebootstrap, so we need to support maintaining some local state for the transaction in this case.

Reply:
Renamed to TransactionLostException for now.